src/Readers/AbstractReader.js

```diff
@@ -40,12 +40,13 @@ class AbstractReader {
    * @return {{__FILE__, __PATH__}}
    */
   parsePath(source) {
-
     const __FILE__ = path.basename(source);
     let __PATH__;
 
     let parsedURL = url.parse(source);
-    if (parsedURL.protocol) {
+
+    // url parse can find protocol in Windows-style path, so we check it
+    if (parsedURL.protocol && !parsedURL.protocol.match(/^(\w\:)?$/i)) {
```

Comment:
I don't think you need the change. As long as you have you slashes substituted `path.dirname` should work pretty good.